### ISSUE TITLE
QoE Metrics Reporting - Name Space Identifier required for DASH QoE metrics reporting parameters?

### ISSUE THREAD

**Issue by dsilhavy** [5G-MAG/Standards#128]
## Description
TS 26.512 Table 7.8.3-1 defines the `metrics` property as a non-empty list of metrics which shall be collected and reported. In the case of downlink media streaming and for the 3GPP scheme `urn:3GPP:ns:PSS:DASH: QM10` the listed metrics shall correspond to the metrics specified in TS 26.247.

Our current client-side implementation uses the following constants to identify which metrics shall be reported by the Media Session Handler to the 5GMS Application Function. Note our implementation does not support all metrics defined in TS 26.247 yet.

````kotlin
object Metrics {
    const val BUFFER_LEVEL = "BufferLevel"
    const val HTTP_LIST = "HTTPList"
    const val REP_SWITCH_LIST = "RepSwitchList"
    const val MPD_INFORMATION = "MPDInformation"
}
````

A corresponding entry in the Service Access Information fetched via M5 looks the following:

````json
{
  "metricsReportingConfigurationId": "7bdfb0b0-fe4a-41ee-8d9e-cb36a16378a2",
  "serverAddresses": [
    "http://192.168.2.7:7778/3gpp-m5/v2/"
  ],
  "scheme": "urn:3GPP:ns:PSS:DASH:QM10",
  "dataNetworkName": "",
  "reportingInterval": 10,
  "samplePercentage": 100,
  "urlFilters": [],
  "samplingPeriod": 5,
  "metrics": [
    "HTTPList",
    "BufferLevel",
    "RepSwitchList",
    "MPDInformation"
  ]
}
````

26.512 Annex E.2.1 defines the reporting parameters for 3GPP-DASH metrics. In the provided examples the identifiers are prefixed with the name space identifier defined as : `urn:3GPP:ns:PSS:DASH:QM10`.

Consequently, to report all metrics of the `AvgThroughput` type the identifier looks like the following `urn:3GPP:ns:PSS:DASH:QM10#AvgThroughput` 

## Discussion / Questions
* Annex E.2.1 does not seem to be referenced anywhere. Should there be a reference to Annex E.2.1 in Table 7.8.3-1
* What is the reason to add the name space identifier to each metric? Isn't it sufficient to provide the identifier via the `scheme` property (see example above)?

**Comment by rjb1000** [comment]
I have looked into this a bit and these are some initial findings:
- When doing QoE metrics reporting for a VR session carried over 5G Media Streaming, TS 26.512 table 7.8.3-1 specifies that the metrics should be as specified in TS 26.118 clause 9.3, and should be reported according to TS 26.118 clause 9.4.
- TS 26.118 clause 9.3 specifies some new metrics for VR. For example, table 9.3.3-1 defines the _Rendered viewports_ metric with fields start time, duration and viewport.
- TS 26.118 clause 9.4 specifies that VR metrics are to be reported using the same basic QM10 schema as DASH metrics, but goes on to specify additional XML Schema types to convey the VR metrics.
  - I interpret this to mean that the provisioned metrics scheme identifier is also therefore `urn:3GPP:ns:PSS:DASH:QM10`, which tells the 5GMS AF what XML document format and MIME type to expect.
- TS 26.532 clause E.2.2 specifies that VR metrics specified in clause 9.4 are to be identified in 5GMS QoE metrics reporting using the namespace _urn:3gpp:metadata:2020:VR:metrics_.
  - For example, the duration field of the _Rendered viewports_ metric would have the fully-qualified term identifier _urn:3gpp:metadata:2020:VR:metrics#RenderedViewports/duration_.
 
So, to make a concrete example of a metrics reporting configuration for VR over 5GMS:
````json
{
  "metricsReportingConfigurationId": "7bdfb0b0-fe4a-41ee-8d9e-cb36a16378a2",
  "serverAddresses": [
    "http://192.168.2.7:7778/3gpp-m5/v2/"
  ],
  "scheme": "urn:3GPP:ns:PSS:DASH:QM10",
  "dataNetworkName": "",
  "reportingInterval": 10,
  "samplePercentage": 100,
  "urlFilters": [],
  "samplingPeriod": 5,
  "metrics": [
    "urn:3gpp:metadata:2020:VR:metrics#RenderedViewports"
  ]
}
````

This explains why the _scheme_ and _metrics_ array members are both fully-qualified term identifiers, even though the namespace is the same for the special case of DASH over 5GMS.

**Comment by rjb1000** [comment]
> Annex E.2.1 does not seem to be referenced anywhere. Should there be a reference to Annex E.2.1 in Table 7.8.3-1.

I agree this needs to be properly cross-referenced.

**Comment by rjb1000** [comment]
Change Request contributed to MBS _ad hoc_ meeting post SA4#127-bis-e:

* TS 26.512
  * Rel-17: **CR0069** "[5GMS3, TEI17] Metrics reporting corrections" in [S4aI240047](https://www.3gpp.org/ftp/TSG_SA/WG4_CODEC/3GPP_SA4_AHOC_MTGs/SA4_MBS/Docs/S4aI240047.zip).
  * Rel-18: _Changes to be handled directly in TS 26.510 V1.2.1._

**Comment by rjb1000** [comment]
New specification versions published following approval of CRs at SA#104:
- TS 26.512
  - Rel-17: [V17.9.1](https://www.3gpp.org/ftp/Specs/archive/26_series/26.512/26512-h91.zip).
  - Rel-18: _Not applicable: affected clause ported to TS 26.510._
